manifests/agent.pp

```diff
@@ -18,6 +18,10 @@
   $pxp_ping_interval     = undef,
 ) {
 
+  File {
+    backup => false
+  }
+
```

Comment:
This is a relatively simple change with rather large performance implications. As we have NOT used file resource changes in prior release-testing, it shouldn't be a huge deal from that perspective.

But if most customers leave this enabled (which I'm assuming they do...but it's still an assumption) we should try to make this configurable, if possible.